src/ACEtk/interpretation/EL03/BindingEnergy.hpp

```diff
@@ -0,0 +1,14 @@
+class BindingEnergy {
+  const Table& table;
+
+public:
+  BindingEnergy( const Table& table ) : table( table ){}
+    
+  auto values() const {
+    const auto length = this->table.data.NXS( 11 );
+    const auto start  = this->table.data.JXS( 11 ) + length;
+    return this->table.data.XSS( start, length )
```

Comment:
BindingEnergy and Occupation are related quantities on the same grid. Maybe we should merge these classes